apps/components-e2e/src/components/chart/selection-area/selection-area.e2e.ts

```diff
@@ -254,3 +254,49 @@ test('should create a range out of the timestamp with shift + left arrow key', a
     .expect(await overlayText.textContent)
     .match(/Jul 31 \d{2}:20 — \d{2}:25/g);
 });
+
+test('should not re-open a timestamp in case of a window resize', async (testController: TestController) => {
+  await testController.resizeWindow(1200, 800);
+  await createTimestamp(
+    { x: 450, y: 100 },
+    chartClickTargets[1],
+    testController,
+  );
+
+  await testController.expect(await timestampSelection.exists).ok();
+  await testController.expect(await overlay.exists).ok();
+
+  await closeOverlay();
+
+  await testController.expect(await timestampSelection.exists).notOk();
+  await testController.expect(await overlay.exists).notOk();
+
+  await testController.resizeWindow(1000, 600);
+
+  await testController.expect(await timestampSelection.exists).notOk();
+  await testController.expect(await overlay.exists).notOk();
+
+  await testController.resizeWindow(1200, 800);
+});
+
+test('should not re-open a range in case of a window resize', async (testController: TestController) => {
+  await testController.resizeWindow(1200, 800);
+  const start = { x: 310, y: 100 };
+  await createRange(520, start);
+  // in case of flakyness that the overlay does not update to the right values
+  await testController.wait(500);
+  await testController.expect(await rangeSelection.exists).ok();
+  await testController.expect(await overlay.exists).ok();
+
+  await closeOverlay();
+
+  await testController.expect(await rangeSelection.exists).notOk();
+  await testController.expect(await overlay.exists).notOk();
+
+  await testController.resizeWindow(1000, 600);
+
+  await testController.expect(await rangeSelection.exists).notOk();
+  await testController.expect(await overlay.exists).notOk();
+
+  await testController.resizeWindow(1200, 800);
+});
```

Comment:
Please add a test that checks when a selection is open and you perform a resize of the window, that the selection is kept and not being closed.

Furthermore, it should select the same data (Check what is displayed in the overlay) 

Please add those tests for the range and the timestamp